controllers/extensionRequests.js

```diff
@@ -171,7 +172,39 @@ const updateExtensionRequest = async (req, res) => {
       }
     }
 
-    await extensionRequestsQuery.updateExtensionRequest(req.body, req.params.id);
+    const promises = [extensionRequestsQuery.updateExtensionRequest(req.body, req.params.id)];
+    // If flag is present, then only create log for change in ETA/reason by SU
+    if (dev === "true") {
```

Comment:
Creating logs in dev mode, whenever SU updates the extension requests be it title, reason, or ETA.